CMSgov/mdct-mcr
# Mdct 1919

## Description

Change permissions allowing only admin users to delete programs.

### How to test

Have a program, log in as an Admin and see that the delete button is still there. Log in as an approver or help desk user and see that the delete button is not there.

### Changed Dependencies


## Code author checklist

## Reviewer checklist (two different people)
